src/locale/properties/layout/layout.en.properties

```diff
@@ -90,7 +90,6 @@ ngOrcid.search=Search...
 ngOrcid.copyright=CC0 license.
 ngOrcid.seeMoreStatics=See more statistics
 layout.understood=Understood
-public-layout.membership=Membership
```

Comment:
I think we still use this property.

Reply:
yes, it was duplicated so the script just remove the second one